docs/cookbook/legacyDateFromDateTime.mjs

```diff
@@ -0,0 +1,26 @@
+import assert from "assert";
+
+/**
+ * Converts a Temporal.DateTime instance into a Date.
+ * This is the inverse of dateTimeFromLegacyDate example
+ *
+ * @param {Temporal.DateTime} dateTime This is a Date instance
+ * @returns {Date} Temporal.DateTime instance
+ */
+function getDateFromDateTime(dateTime) {
+    return new Date(
+        dateTime.year,
+        dateTime.month - 1,
+        dateTime.day,
+        dateTime.hour,
+        dateTime.minute,
+        dateTime.second,
+        dateTime.millisecond
+    );
+}
+
+const dateTime = Temporal.DateTime.from("1970-01-01T00:00:01Z");
+const result = getDateFromDateTime(dateTime);
+assert.equal(result.getDate(), 1); // Day of the month
+assert.equal(result.getFullYear(), 1970);
+assert.equal(result.getMonth(), 0); // the month (zero-indexed)
```

Comment:
I think it would be nice to add a concise assertion of all the data, e.g. `assert.equal(result.toISOString(), "1970-01-01T10:00:01.000Z")`.

Reply:
This fails :/

Reply:
``` 
 actual: '1969-12-31T23:00:01.000Z',
  expected: '1970-01-01T10:00:01.000Z',
  operator: '=='
```

Reply:
Good point. This will have a different result based on your system's current time zone. We should rename the function to something like `getLegacyDateInLocalTimeZoneFromDateTime()`. I think the only other consistent thing we could do instead, is use `Date.UTC()` instead of `new Date()` and rename the function to something like `getLegacyDateInUTCFromDateTime()`.